data/roadmap/roadmap.yaml

```diff
@@ -48,133 +48,92 @@ categories:
     things:
       -
         status: Done
-        thing: "18F Phase I: Kickoff & Discovery"
+        thing: "18F Phase I: Discovery"
       -
         status: Done
-        thing: "18F Phase I: User Research"
+        thing: "18F Phase II: Search and Filter mapper alpha development"
       -
-        status: Done
-        thing: "18F Phase I: Synthesis & prioritization"
-      -
-        status: Done
-        thing: "18F Phase I: Roadmap & Low-fidelity prototyping"
+        status: on deck
+        thing: "Search and Filter mapper beta development"
       -
-        status: Done
-        thing: "18F Phase II: Planning"
+        status: on deck
+        thing: "Consolidated download beta development"
       -
-        status: "In progress"
-        thing: "18F Phase II: Prototype"
-      -
-        status: "In progress"
-        thing: "18F Phase II: Iterate"
-      -
-        status: "On deck"
-        thing: "18F Phase II: Transition & Handoff"
+        status: upcoming
+        thing: "Decommissioning NWISmapper"
   -
     name: "Next-generation monitoring location pages"
     status: "in progress"
     things:
-      -
-        status: in progress
-        thing: "Usability testing"
-      -
-        status: in progress
-        thing: "Implement new date range picker and tooltips"
-      - status: in progress
-        thing: "Use USGS National Map background layers"
-      - status: done
-        thing: "Put AHPS flood levels on gage height hydrograph"
       -
         status: done
-        thing: "Compile user feedback and prioritize new features and usability improvements"
+        thing: "Site-specific alerts (info, warning, errors)"
```

Comment:
Is this Water Alert?

Reply:
I'll add some clarification to this entry and the similar one right after that will say "Dynamic site-specific <whatever it is> that can be added by Water Science Center staff"